src/sqlparser/src/ast/mod.rs

```diff
@@ -2187,6 +2171,36 @@ impl fmt::Display for Statement {
     }
 }
 
+impl Display for IncludeOptionItem {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let Self {
+            column_type,
+            inner_field,
+            header_inner_expect_type,
+            column_alias,
+        } = self;
+        write!(f, "INCLUDE {}", column_type)?;
+        if let Some(inner_field) = inner_field {
+            write!(f, " '{}'", value::escape_single_quote_string(inner_field))?;
+            if let Some(expected_type) = header_inner_expect_type {
+                write!(
+                    f,
+                    " {}",
+                    match expected_type {
+                        DataType::Varchar => "varchar",
+                        DataType::Bytea => "bytea",
+                        t => unreachable!("unparse header expected type: {t}"),
+                    }
```

Comment:
Will refactor the data type logic further in a follow-up PR. This one intends to minimize the changes to parsing logic and focus on `Display`. There is a lot of space to improve for the parsing & handling of data type as well.